README.md

```diff
@@ -1 +1,40 @@
-# js-mentoring-program
\ No newline at end of file
+# JavaScript Mentoring Program
+
+## Installation
+
+To run the project you should have some tools installed on your PC, which you can find in a list of links below:
+
+- [Download and install Git](https://git-scm.com/downloads)
+- [Download and install](https://nodejs.org/en/) latest LTS-version of Node.js
+- Run `node -v` in console and make sure your version of Node.js is above `v8.11.3`
```

Comment:
Could you update Node version to 12 or 14 (and comment about the version)?
If you need to use different Node versions for different projects, you can use `nvm` (https://github.com/nvm-sh/nvm) for changing Node version for each project.